Simplenote/src/androidTest/java/com/automattic/simplenote/utils/EspressoUtils.kt

```diff
@@ -0,0 +1,44 @@
+package com.automattic.simplenote.utils
+
+import android.view.View
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+fun hasTextInputLayoutHintText(expectedErrorText: String): Matcher<View> {
+    return object : TypeSafeMatcher<View>() {
+        override fun matchesSafely(view: View): Boolean {
+            if (view !is TextInputLayout) {
+                return false
+            }
+
+            val error = (view as TextInputLayout).hint ?: return false
+            val hint = error.toString()
+
+            return expectedErrorText == hint;
+        }
+        override fun describeTo(description: Description) {
+
+        }
+    }
+}
+
+fun hasTextInputLayoutErrorText(expectedErrorText: String): Matcher<View> {
+    return object : TypeSafeMatcher<View>() {
+        override fun matchesSafely(view: View): Boolean {
+            if (view !is TextInputLayout) {
+                return false
+            }
+
+            val error = (view as TextInputLayout).error ?: return false
```

Comment:
Minor (🔍): This cast to `TextInputLayout` is redundant, isn't it. Consider simplifying this line.

PS: Same applies to the function above.

Reply:
Yes, it is useless. Removed.